src/web/mjs/connectors/Manga9.mjs

```diff
@@ -0,0 +1,69 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Manga9 extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'manga9';
+        super.label = 'Manga Raw (manga9.co)';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://manga9.co';
+
+        this.path = '/list/?page=';
+        this.queryMangaTitle = 'main#primary h1';
+        this.queryMangaPages = 'div.wp-pagenavi span.pages';
+        this.queryMangas = 'div.card-body div.rotate-img a';
+        this.queryChapters = 'div.list-scoll a';
+        this.queryPages = 'div.card-wrap source';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, this.queryMangaTitle);
+        const id = uri.pathname;
+        const title = data[0].textContent.replace('(Raw – Free)', '').trim();

```

Comment:
May use regex in case there are chapters witout `Free` keyword `/\(Raw( - Free)?\)/`

Reply:
Fixed